package.json

```diff
@@ -44,7 +44,9 @@
     "jsonc-parser": "^3.2.0",
     "lines-and-columns": "^2.0.3",
     "nanospinner": "^1.1.0",
-    "semver": "^7.5.4"
+    "semver": "^7.5.4",
+    "fs-extra": "^11.0.1",
+    "ignore": "^5.2.4"
```

Comment:
Any reason to add thos two dependencies ?

Reply:
Both of these dependencies are getting used in `gitignore.ts` but there is no mention of these two in the `package.json`. As a result, while running the command line utility it throwing error as `Can't find module 'ignore'` and `Can't find module 'fs-extra'`.